src/main/java/core/db/DBUpdater.java

```diff
@@ -1189,21 +1189,23 @@ private boolean columnExistsInTable(String columnName, String tableName) throws
 				+ columnName.toUpperCase()
 				+ "'";
 		ResultSet rs = this.m_clJDBCAdapter.executeQuery(sql);
-		return rs.next();
+		if ( rs != null )return rs.next();
+		return false;
 	}
 
 	private boolean tableExists(String tableName) throws SQLException {
 		String sql = "SELECT * FROM INFORMATION_SCHEMA.SYSTEM_TABLES WHERE TABLE_NAME = '"
 				+ tableName.toUpperCase() + "'";
 		ResultSet rs = this.m_clJDBCAdapter.executeQuery(sql);
-		return rs.next();
-	}
+		if ( rs != null )return rs.next();
+		return false;	}
 
 	private boolean indexExists(String indexName, String tableName) throws SQLException {
 		String sql = "SELECT * FROM INFORMATION_SCHEMA.SYSTEM_INDEXINFO WHERE INDEX_NAME = '"
 				+ indexName.toUpperCase() + "' AND TABLE_NAME = '" + tableName.toUpperCase() + "'";
 		ResultSet rs = this.m_clJDBCAdapter.executeQuery(sql);
-		return rs.next();
```

Comment:
is this due to a change in java 14 ?  before `rs.next()` was returning false and now he throws an error if rs is null ??
Is this why you had to change it everywhere ?

Reply:
git commit asked me to review the code and gives warnings that rs could be null